package.json

```diff
@@ -4,11 +4,14 @@
   "description": "A set of primitives to build simple, flexible, WAI-ARIA compliant React autocomplete components",
   "main": "dist/downshift.cjs.js",
   "jsnext:main": "dist/downshift.esm.js",
+  "react-native": "dist/downshift.native.cjs.js",
   "module": "dist/downshift.esm.js",
   "typings": "typings/index.d.ts",
   "scripts": {
     "add-contributor": "kcd-scripts contributors add",
-    "build": "kcd-scripts build --bundle --p-react",
+    "build": "npm run build:web --silent && npm run build:native --silent -- --no-clean",
+    "build:web": "kcd-scripts build --bundle --p-react",
+    "build:native": "BUILD_REACT_NATIVE=true BUILD_FILENAME_SUFFIX=.native kcd-scripts build --bundle cjs",
```

Comment:
Let's add a devDependency on `cross-env` and use that here so this works on windows machines as well (considering the `setup` script will run `validate` which will run the build. I want to make sure that script always works on windows.

Reply:
done